Samples/resource-owner-password-credential/Angular2/Angualar2-Client-ROPC/app/boot.ts

```diff
@@ -0,0 +1,19 @@
+import { bootstrap } from '@angular/platform-browser-dynamic';
+import { HTTP_PROVIDERS } from '@angular/http';
+import { APP_ROUTER_PROVIDERS } from './app.route';
+import { AppComponent } from './app.component';
+import { Configuration } from './app.constants';
+import { enableProdMode } from '@angular/core';
+import {JwtHelper} from 'angular2-jwt'
+import {Authservice} from './authorize/authorize-service';
+import {ResourceService} from './resource/resource-service';
+import {disableDeprecatedForms, provideForms} from '@angular/forms';
```

Comment:
Are there any reasons to make this module import case mixed? e.g `import { Configuration }` vs `import {JwtHelper}`